content/docs/command-reference/plots/index.md

```diff
@@ -126,7 +261,7 @@ DVC has the following built-in plot templates:
 
 - `linear` - basic linear plot including cursor interactivity (default)
 - `simple` - simplest linear template (not interactive); Good base to create
-  [custom templates].
+  [custom template].
 - `scatter` - scatter plot
 - `smooth` - linear plot with LOESS smoothing, see
   [example](/doc/command-reference/plots#example-smooth-plot)
```

Comment:
Should be plural (1/2)
```suggestion
  [custom templates].
- `scatter` - scatter plot
- `smooth` - linear plot with LOESS smoothing, see
  [example](/doc/command-reference/plots#example-smooth-plot)
```

Reply:
Shoot I can't easily fix the actual link in line 271... (should be `[custom templates]` there)... ⏳

Reply:
OK it's time for me to clone this fork 😬 ...

Reply:
Fixed.